engine/src/main/java/org/terasology/rendering/dag/nodes/WorldReflectionNode.java

```diff
@@ -199,4 +215,37 @@ public void process() {
 
         PerformanceMonitor.endActivity();
     }
+
+    @Override
+    public void propertyChange(PropertyChangeEvent event) {
```

Comment:
This method seem to be an exact duplicate of the same method in another two classes: to be obtained via inheritance instead?